.github/workflows/wheels.yml

```diff
@@ -0,0 +1,103 @@
+name: Build CI Wheels
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'pyart-*'
+  pull_request:
+    paths:
+      - '.github/workflows/wheels.yml'
+  workflow_dispatch:
+
+env:
+  CIBW_TEST_REQUIRES: pytest
+  CIBW_TEST_COMMAND: "pytest {project}/tests"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [
+          ubuntu-20.04,
+          windows-2019,
+          macos-11,
+        ]
+      fail-fast: false
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      
+      - name: Fetch all history for all tags and branches
+        run: |
+           git fetch --prune --unshallow
+
+      - name: Install netcdf4
+        run: python -m pip install netCDF4
+
+      - name: Build wheels for CPython
+        uses: pypa/cibuildwheel@v2.9.0
+        with:
+          output-dir: dist
+        env:
+          CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-* cp310-*"
```

Comment:
I suggest including wheels for Python 3.11 while you're at it. The supply chain is ready for this (CIBW supports it and CPython 3.11 has reached ABI stability).
```suggestion
          CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-* cp310-* cp311-*"
```

Reply:
on a related topic, I'd suggest specifying `python_requires=">=3.6.1"` in `setup.py`, so that a reasonable error is raised when trying to build from source with older versions of Python